### PR TITLE
Bump RL9 host image to RL9.3

### PR DESCRIPTION
**Pull request by MoteHue**

### CODE REVIEW

**Review by markgoddard — DISMISSED** [review]

**Comment by jovial** [comment]
I've updated as CI job wasn't running (hoping this means we pick up the recent changes to CI)

**Comment by markgoddard** [comment]
Rocky 9 jobs failed because the image wasn't uploaded to Leafcloud. I've run the [new workflow](https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/7974702265) to do this.

Upgrade jobs failed because the ci-aio env in yoga was updated to disable heat, but it is still enabled in zed, so DB migrations fail. I'll merge through the relevant changes to zed & antelope.

**Comment by markgoddard** [comment]
I've changed the target branch, hopefully it will pass now. Please wait for #950 to merge before merging this though.

**Review by markgoddard — APPROVED** [review]

**Pull request merged**
